packages/fixie/src/agent.ts

```diff
@@ -87,12 +87,7 @@ export class FixieAgent extends FixieAgentBase {
       `
     );
     const program = TJS.programFromConfig(tsconfigPath, [tempPath]);
-    const schema = TJS.generateSchema(program, 'RuntimeParameters', settings);
-    if (schema && schema.type !== 'object') {
-      throw new Error(`The first argument of your default export must be an object (not ${schema.type})`);
-    }
-
-    return schema;
+    return TJS.generateSchema(program, 'RuntimeParameters', settings);
```

Comment:
Is this check not still relevant?

Reply:
It's overly restrictive and precludes `oneOf`/`allOf`. The actual requirement is that an empty object satisfies the schema, which the service verifies.